### DISCUSSION TITLE
Client work through bluetooth

### DISCUSSION THREAD

**Discussion by bbest1** [MusicPlayerDaemon/MPD#1726] [General]
Hello!
I have server with installed MPD + USB output DSD, wav files and so on. There are no network, no WiFi, no video card, only pure Linux with MPD + alsa + blutooth module and USB output.
Is there exists some client which could work on smartphone and control MPD server through bluetooth?

**Comment by skidoo23** [comment]
https://wiki.archlinux.org/title/android_tethering#Tethering_via_Bluetooth

**Comment by bbest1** [comment]
got it, thank you.
I wanted to use this client: https://jcorporation.github.io/myMPD/ , light and functional.

**Comment by skidoo23** [comment]
I really like myMPDs webradiodb / radio-browser feature. But I prefer MALP https://gitlab.com/gateship-one/malp over a web based client. YMMV.

**Comment by bbest1** [comment]
Thank you for advice! On PC I will use Web client myMPD and on Android may be MALP.

**Comment by mikwei** [comment]
I setup a web-server on MPD server.  The I can access from phone, tablet...  Many music players support MPD, but you need to google.  Don't know your setup, but I use Linux and is very easy.
